cupy/core/include/cupy/atomics.cuh

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#if __CUDA_ARCH__ < 600
```

Comment:
Please check this condition.
I got some error on CUDA7 environment.
```
/tmp/tmp4Owu_E/kern.cu(8): error: function "atomicAdd(double *, double)" has already been defined
```

Reply:
As @kmaehashi mentioned, there are conflicts on declaration of double precision `atomicAdd`. I will fix it.

Reply:
OK.